mne/channels/interpolation.py

```diff
@@ -95,13 +74,17 @@ def _make_interpolation_matrix(pos_from, pos_to, alpha=1e-5):
     cosang_from = pos_from.dot(pos_from.T)
     cosang_to_from = pos_to.dot(pos_from.T)
     G_from = _calc_g(cosang_from)
-    G_to_from, H_to_from = (f(cosang_to_from) for f in (_calc_g, _calc_h))
+    G_to_from = _calc_g(cosang_to_from)
 
     if alpha is not None:
         G_from.flat[::len(G_from) + 1] += alpha
 
-    C_inv = linalg.pinv(G_from)
-    interpolation = G_to_from.dot(C_inv)
+    C = np.r_[np.c_[G_from, np.ones((G_from.shape[0], 1))],
+              np.c_[np.ones((1, G_from.shape[1])), 0]]
+    C_inv = linalg.pinv(C)
```

Comment:
this is a change although valid.

you should document it as a BUG fix...

@dengemann ok for you?

maybe @wmvanvliet has an opinion too ...